data/Bactrian_Search.txt

```diff
@@ -0,0 +1,1326 @@
+				accept
+			label piratejoke2
+			`	She says, "Hahaha, I am starting to like you, okay 2.8 million and it's all yours."`
+			choice
+				`	"You are so generous...now where was the expedition headed?"`
+				`	"That's a terrible discount."`
+			`	"Go to the Mebutsa system, you will have your data and be refueled within the hour." As she walks away she slides into a piratey slang yelling at her subordinates with ar's and mates. It seems she enjoys being the pirate captain a bit.`
+				accept
+	on enter Almaaz
+		event "pirate moon base lock"
+	on fail
+		dialog `You failed to convince the pirates to share thier information, your only chance is to go to the library at Alexandria.`
+	on accept
+		fail "epbconvo"
+		set epicpiratebattleNO
+	on complete
+		fail "bactrian search: 2a"
+		set "bacsearch 2"
+
+ship "Hepatica" "Modified Hepatica"
+	outfits
+		"Electron Beam" 2
+		"Hai Tracker Pod" 4
+		"Hai Tracker" 400
+		"Electron Turret" 4
+		"Heavy Anti-Missile Turret" 2
+		"Plasma Turret" 2
+		"Triple Plasma Core"
+		"Systems Core (Large)"
+		"Large Heat Shunt" 4
+		"Small Heat Shunt"
+		"Outfits Expansion" 2
+		
+		`"Bondir" Atomic Thruster`
+		"Steering (Stellar Class)"
+	
+	gun "Hai Tracker Pod"
+	gun "Hai Tracker Pod"
+	gun "Electron Beam"
+	gun "Electron Beam"
+	gun "Hai Tracker Pod"
+	gun "Hai Tracker Pod"
+	turret "Electron Turret"
+	turret "Electron Turret"
+	turret "Electron Turret"
+	turret "Electron Turret"
+	turret "Heavy Anti-Missle Turret"
+	turret "Heavy Anti-Missle Turret"
+	turret "Plasma Turret"
+	turret "Plasma Turret"
+
+mission epblanding
+	name "Defeat the pirate fleet"
+	clearance
+	landing
+	source "Pirate Moon Base"
+	destination "Pirate Moon Base"
+	to offer
+		has epicpiratebattle
+		has "bacsearch 1: done"
+		not "bactrian search 2a: done"
+	npc kill
+		system Unagi
+		government "Northern Pirates"
+		personality heroic nemesis staying
+		ship "Modified Hepatica" "Eye of Horus" tag nocapture
+		ship "Marauder Raven" "Blade of Anhur"
+		ship "Marauder Raven" "Arrow of Anat"
+		ship "Marauder Raven" "Apedemak's Roar"
+		ship "Marauder Raven" "Blubber of Montu"
+		ship "Marauder Raven" "Beak of Nekhbet"
+		ship "Marauder Raven" "Reshep the Weak"
+		ship "Marauder Raven" "Yelp of Wepwawet"
+		ship "Osprey" "Beard of Imsety" tag nocapture
+		ship "Osprey" "Cackle of Duamutef" tag nocapture
+		ship "Osprey" "Shriek of Hapi" tag nocapture
+		ship "Osprey" "Gaze of Qebehsenuef" tag nocapture
+		ship "Marauder Falcon" "Hound of Anubis" tag nocapture
+		ship "Marauder Falcon" "Song of Hathor" tag nocapture
+		dialog "Now you should be able to look through the base"
+	on offer
+		conversation
+			`	On your landing approach a pirate fleet launches from the surface. Leroy your co-pilot swears something about being ducks in a puddle while you change course to deal with the pirates.`
```

Comment:
Would it sound less like a 4th wall breach like this:
`	Suddenly, a pirate fleet launches from the surface. Leroy your co-pilot swears something about being ducks in a puddle and changes course to deal with the pirates.`

Reply:
(apart from a big Bactrian mission string PR that's been worked on for very long and that it's way more likely to be merged which makes working on this basically redundant)
remove that second line entirely and rewrite it to something without mentioning a co-pilot, ducks, or a puddle

Reply:
The player might have a ship that only takes one person to fly.

Reply:
hence the "without mentioning a co-pilot", but good thing ya specified the reason

Reply:
Since this only occurs after an event requiring disabling a modified argosy, it would be highly unlikely (read impossible) that anyone would be in a single crew ship.

Reply:
capturing MMK or using any other 1-man ship, and have big-ass escorts

Reply:
Never make any unneeded assumptions for the player. Ever.

Reply:
Should I remove the "bactrian search: 2b special" mission also so as to not assume pirate relationship?
I am truly wanting an honest answer.